Week3/BusinessCard/app/src/main/java/com/raywenderlich/businesscard/nivaldolopo/MainActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.raywenderlich.businesscard.nivaldolopo
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.animation.AnimationUtils
+import android.widget.Button
+import androidx.appcompat.app.AlertDialog
+
+class MainActivity : AppCompatActivity() {
+
+    internal lateinit var buttonQRCode: Button

```

Comment:
If you can, avoid lateinit vars, and use Kotlin Android Extensions or ViewBinding!